Dandelion Sprout's Anti-Malware List.txt

```diff
@@ -1682,6 +1682,24 @@ zombooru.com##a[href="http://www.hard55.com"]
 ||dl02.s3.amazonaws.com/installers/747947/oi_picasa38-setupexe.exe^$all
 ! https://www.virustotal.com/gui/file/2ea599605c4d65902943f12e1114a71af7a40fa7dffbf018b0ee3e7a61aaeaa3/community
 ||dl02.s3.amazonaws.com/installers/424531/2gzbsoj4gxb.exe^$all
+! https://www.huorong.cn/info/1531309921141.html
+||kuaizip.com^$all
+! https://www.huorong.cn/info/1618397948649.html
+! ||zhuangjizhuli.com^$all
+! ||zhuangjizhuli.net^$all
+! https://github.com/uBlockOrigin/uAssets/pull/9656
+||geekotg.com^$all
+||qq789.com.cn^$all
+! https://www.huorong.cn/info/1526627586130.html
+||xiaobaixitong.com^$all
+! https://www.huorong.cn/info/1577158839403.html
+||daque.cn^$all
+! https://www.huorong.cn/info/1598957552515.html
+||dabaicai.com^$all
+! https://www.huorong.cn/info/1617368984641.html
+||qqfzn.com^$all
+! https://github.com/uBlockOrigin/uAssets/pull/10017
```

Comment:
Is this still online?

Reply:
Uh, I don't quite understand what you mean. Does you mean `flash.cn`?

Reply:
Yes.

Reply:
With a US VPN I just get a valid signed-by-Adobe version. Will try with China VPN

Reply:
Wait, yep, there is some shady stuff going on...

Reply:
![image](https://user-images.githubusercontent.com/84232764/144315859-18cb9d92-ba32-48bd-b78c-340ae04a0d43.png)

Reply:
Some kind of fake IE?
![image](https://user-images.githubusercontent.com/84232764/144316210-6e90f0dc-51a6-4631-8a62-a17cfcc68a02.png)

Reply:
The malware shutdown my shared folder (how rude), so I had to hack together an implementation using TeamViewer, which is why it is running:
![flashm](https://user-images.githubusercontent.com/84232764/144317807-d0deda7f-d65f-461e-9e2c-b0cd15fc7156.jpeg)

Reply:
A lot of the files dropped by the malware are unknown to VT. Attached a zip of all of them for those interested. 

https://github.com/iam-py-test/investigations/blob/main/samples/malware.zip